apps/ivrs/api_views.py

```diff
@@ -39,7 +39,8 @@ def get(self, request):
         parameters['raw_data'] = request.QUERY_PARAMS.get('Body', None)
         parameters['telephone'] = request.QUERY_PARAMS.get('From', None)
         parameters['session_id'] = request.QUERY_PARAMS.get('SmsSid', None)
-        parameters['telephone'] = parameters['telephone'][1:] # Strip 0
+        if parameters['telephone'] != None:
```

Comment:
A cleaner & more pythonic way would be to just write `if parameters['telephone']:`